packages/tool/site/server/src/routes/sign/uploadSignatures.ts

```diff
@@ -26,11 +27,13 @@ export default [
       const prepareResponses = successfulPreparePrescriptionIds.map((id: string) => {
         return {
           id: id,
-          response: getSessionValue(`prepare_response_${id}`, request)
+          response: getSessionValue(`prepare_response_${id}`, request),
+          signingOptions: signInHeaders
```

Comment:
I would prefer this to be a separate parameter on uploadSignatureRequest

Reply:
this has been taken out.